operations/node_management/Rebuild_NCNs.md

```diff
@@ -666,6 +666,16 @@ This section applies to master and worker nodes. Skip this section if rebuilding
            systemctl start etcd.service; /srv/cray/scripts/common/kubernetes-cloudinit.sh
    ```
 
+    **Rebuilt node with modified ssh key(s):** The cloud-init process can fail when accessing other nodes if ssh keys have been modified in the cluster.  If this occurs, the following steps can be used to repair the desired ssh keys on the newly rebuilt node:
+
+    1. Allow cloud-init fail due to the non-matching keys.
```

Comment:
"cloud-init fail" -> "cloud-init to fail"

Reply:
great catch -- fixing

Reply:
fixed